### PR TITLE
[Sprint 6]Feature ui manager in shop

### PR DESCRIPTION
**Pull request by nguyenhuy95dn**
Feature ui manager in shop
Link redmine : https://dev.framgia.com/redmine/issues/121282
Task : Domain management
Effect: N/A

### CODE REVIEW

File: app/src/main/java/com/framgia/forder/data/model/OwnerShop.java [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
 import com.google.gson.annotations.Expose;
 import com.google.gson.annotations.SerializedName;
 
 /**
  * Created by Age on 6/15/2017.
  */
 
-public class OwnerShop {
+public final class OwnerShop implements Parcelable {
+    public static final Creator<OwnerShop> CREATOR = new Creator<OwnerShop>() {
```
Comment (Tuanlvt) [inline app/src/main/java/com/framgia/forder/data/model/OwnerShop.java]: add a blank line

File: app/src/main/java/com/framgia/forder/screen/managerinshop/ItemManagerInShopViewModel.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+public class ItemManagerInShopViewModel extends BaseObservable {
+    private final OwnerShop mOwnerShop;
+
+    ItemManagerInShopViewModel(OwnerShop ownerShop) {
+        mOwnerShop = ownerShop;
+    }
+
+    public String getManagerImage() {
+        //To do fix later when fix api
```
Comment (HyperionsThs) [inline app/src/main/java/com/framgia/forder/screen/managerinshop/ItemManagerInShopViewModel.java]: xóa luôn để làm gì nữa

File: app/src/main/java/com/framgia/forder/screen/managerinshop/ManagerInShopFragment.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+import com.framgia.forder.data.model.OwnerShop;
+import com.framgia.forder.databinding.FragmentManagerInShopBinding;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Managerinshop Screen.
+ */
+public class ManagerInShopFragment extends DialogFragment {
+    private static final String EXTRA_OWNER = "EXTRA_OWNER";
```
Comment (HyperionsThs) [inline app/src/main/java/com/framgia/forder/screen/managerinshop/ManagerInShopFragment.java]: enter kìa

File: app/src/main/java/com/framgia/forder/screen/managerinshop/ManagerInShopPresenter.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
+package com.framgia.forder.screen.managerinshop;
+
+/**
+ * Listens to user actions from the UI ({@link ManagerInShopFragment}), retrieves the data and
+ * updates
+ * the UI as required.
+ */
+final class ManagerInShopPresenter implements ManagerInShopContract.Presenter {
+    private static final String TAG = ManagerInShopPresenter.class.getName();
```
Comment (HyperionsThs) [inline app/src/main/java/com/framgia/forder/screen/managerinshop/ManagerInShopPresenter.java]: enter

Comment (nguyenhuy95dn) [inline app/src/main/java/com/framgia/forder/screen/managerinshop/ManagerInShopPresenter.java]: template là z

File: app/src/main/java/com/framgia/forder/screen/managerinshop/ItemManagerInShopViewModel.java [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        return "http://res.cloudinary.com/hoangmirs/image/upload/c_fill,g_north,h_100,"
```
Comment (Tuanlvt) [inline app/src/main/java/com/framgia/forder/screen/managerinshop/ItemManagerInShopViewModel.java]: Do not write it like this!

File: app/src/main/java/com/framgia/forder/screen/managerinshop/ManagerInShopFragment.java (hunk 3 above)
Comment (Tuanlvt) [inline app/src/main/java/com/framgia/forder/screen/managerinshop/ManagerInShopFragment.java]: add a blank line

File: app/src/main/java/com/framgia/forder/screen/managerinshop/ManagerInShopPresenter.java (hunk 4 above)
Comment (Tuanlvt) [inline app/src/main/java/com/framgia/forder/screen/managerinshop/ManagerInShopPresenter.java]: add a blank line

**Comment by Tuanlvt** [comment]
Please check comment!

File: app/src/main/java/com/framgia/forder/screen/managerinshop/ItemManagerInShopViewModel.java [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+import com.framgia.forder.data.model.OwnerShop;
+
+/**
+ * Created by nguyenhuy95dn on 6/25/2017.
+ */
+
+public class ItemManagerInShopViewModel extends BaseObservable {
+
+    private static final String DEFAULT_AVATAR_LINK =
+            "http://res.cloudinary.com/hoangmirs/image/upload/c_fill,g_north,h_100,"
```
Comment (Tuanlvt) [inline app/src/main/java/com/framgia/forder/screen/managerinshop/ItemManagerInShopViewModel.java]: tách 2 link ra

**Pull request merged**